builtin/game/falling.lua

```diff
@@ -243,14 +256,26 @@ function core.check_single_for_falling(p)
 				core.get_node_max_level(p_bottom))) and
 
 				(not d_bottom.walkable or d_bottom.buildable_to) then
-			convert_to_falling_node(p, n)
-			return true
+			return convert_to_falling_node(p, n)
```

Comment:
The current state of the code has a second call to `convert_to_falling_node(p, n)`, on merge git will only apply one change.
So please rebase.